python/fb_ads_library_api.py

```diff
@@ -80,8 +93,20 @@ def _get_ad_archives_from_url(
         start_time_cutoff_after = datetime.strptime(after_date, "%Y-%m-%d").timestamp()
 
         while next_page_url is not None:
+            print(">> requesting page at " + next_page_url)
             response = requests.get(next_page_url)
             response_data = json.loads(response.text)
+            print(">> got response!")
+
+            # get rate limiting details from headers
+            usage = response.headers.get("x-business-use-case-usage")
```

Comment:
[Am I reading this incorrectly](https://developers.facebook.com/docs/graph-api/overview/rate-limiting/) or is this header only set for API versions up to 3.3?

Reply:
That seems like an accurate reading to me, but I am definitely receiving `x-business-use-case-usage` headers on v20.0. I added checks for the other two headers mentioned on that page in 77497f040ae02973c950cd98acf85d6cc27820f0